.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, "3.10", 3.11]
+        python-version: [ 3.11, 3.12, 3.13]
```

Comment:
dropping 3.10?

Reply:
> Oh zarr 3.0.6 only supports python 3.11+ so this would also need to drop 3.10

Reply:
notably this is just for zarr reading. bioio will still be 3.10-3.13